orangecontrib/text/widgets/icons/Twitter.svg

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
I like, is nice. But I agree with the size thing, since we're nitpicking here. :grin:  I think the circumference of the icon should be the same as the one of GEOmap. :earth_africa: